wildlifelicensing/apps/applications/views/entry.py

```diff
@@ -515,13 +562,13 @@ def get_context_data(self, **kwargs):
             return redirect('wl_applications:new_application')
 
         kwargs['is_payment_required'] = not is_licence_free(generate_product_title(application)) and \
-            not application.invoice_reference and is_customer(self.request.user)
+                                        not application.invoice_reference and is_customer(self.request.user)
```

Comment:
You only do this sort of indenting when the code is within parentheses of some sort. Same for below on line 641